### ISSUE TITLE
Conflict with hardhat-ethers when running tsc

### ISSUE THREAD

**Issue by tuler** [wighawag/hardhat-deploy#51]
When we run `tsc` we get these errors:

```
node_modules/@nomiclabs/hardhat-ethers/dist/src/type-extensions.d.ts:6:10 - error TS2300: Duplicate identifier 'Libraries'.

6     type Libraries = LibrariesT;
           ~~~~~~~~~

  node_modules/hardhat-deploy/dist/src/type-extensions.d.ts:261:10
    261     type Libraries = {
                 ~~~~~~~~~
    'Libraries' was also declared here.

node_modules/hardhat-deploy/dist/src/type-extensions.d.ts:261:10 - error TS2300: Duplicate identifier 'Libraries'.

261     type Libraries = {
             ~~~~~~~~~

  node_modules/@nomiclabs/hardhat-ethers/dist/src/type-extensions.d.ts:6:10
    6     type Libraries = LibrariesT;
               ~~~~~~~~~
    'Libraries' was also declared here.


Found 2 errors.
```

**Comment by wighawag** [comment]
Thanks for the report,
fixed in  0.7.0-beta.21
https://github.com/wighawag/hardhat-deploy/blob/master/CHANGELOG.md#070-beta21

**Comment by tuler** [comment]
I still see the error below when doing `tsc`.

```
node_modules/hardhat-deploy/dist/src/type-extensions.d.ts:3:47 - error TS2307: Cannot find module '../types' or its corresponding type declarations.

3 import { Address, DeploymentsExtension } from '../types';
                                                ~~~~~~~~~~


Found 1 error.

error Command failed with exit code 2.
```

**Comment by tuler** [comment]
Looks like it should be this.

```diff
- import { Address, DeploymentsExtension } from '../types';
+ import { Address, DeploymentsExtension } from './types';
```

**Comment by wighawag** [comment]
Hopefully fixed in 0.7.0-beta.22
